storage/cloudspanner/log_storage.go

```diff
@@ -682,7 +688,7 @@ func (tx *logTX) populateLeafData(ctx context.Context, byHash leavesByHash) erro
 		keySet = append(keySet, spanner.Key{tx.treeID, []byte(k)})
 	}
 	cols := []string{colLeafIdentityHash, colLeafValue, colExtraData, colQueueTimestampNanos}
-	rows := tx.stx.Read(ctx, leafDataTbl, spanner.KeySets(keySet...), cols)
+	rows := tx.stx.Read(ctx, leafDataTbl, spanner.KeySets(keySet...), cols) // @@@@
```

Comment:
@@@@ ?

Reply:
That was a reminder to come back and check whether the signature is needed in this path. It isn't -- this path only populates a `LogLeaf` not a `QueuedLogLeaf`